pandas/core/frame.py

```diff
@@ -3944,15 +3966,17 @@ def _sanitize_column(self, value):
         return value
 
     @property
-    def _series(self):
+    def _series(self) -> Dict[int, Series]:
```

Comment:
`item` in `idx, item in enumerate(self.columns)` is not necessarily an `int`

Reply:
Right, should be `Label` I think